### ISSUE TITLE
Canvas black space

### ISSUE THREAD

**Issue by ThePhilgrim** [ThePhilgrim/minesweeper#29]
Since last last merge, I get a black space between the playing area and the sidebar. Do we have some kind of minimum width of the root window?

![Screenshot 2021-01-21 at 23 34 41](https://user-images.githubusercontent.com/76887896/105421940-3663b380-5c43-11eb-99e4-2eb596bcff73.png)

**Comment by Akuli** [comment]
I went back in git history with `git checkout`. The problem appeared between e4852b1 and b2751cc. To see what code changes there were, I ran `git diff e4852b1 b2751cc`. It seems like the status bar is too wide.

To fix this, a width shouldn't be specified for the middle part of the status bar. Instead, the status bar parts should be packed so that they stretch properly. This can be done by e.g. making a frame for all status bar parts.
